### PR TITLE
build: improve api report

### PR DESCRIPTION
**Pull request by Yeti-or**
:baby_chick: Download canary assets:

[colors--canary.349.4193109950.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.349.4193109950.xml)  
[color_metro_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.349.4193109950.swift)  
[color_metro_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.349.4193109950.kt)  
[color_metro_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.349.4193109950.ts)  
[color_sbermarket_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.349.4193109950.swift)  
[color_sbermarket_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.349.4193109950.kt)  
[color_sbermarket_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.349.4193109950.ts)  
[color_sberprime_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.349.4193109950.swift)  
[color_sberprime_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.349.4193109950.kt)  
[color_sberprime_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.349.4193109950.ts)  
[color_selgros_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.349.4193109950.swift)  
[color_selgros_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.349.4193109950.kt)  
[color_selgros_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.349.4193109950.ts)  
[color_smbusiness_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.349.4193109950.swift)  
[color_smbusiness_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.349.4193109950.kt)  
[color_smbusiness_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.349.4193109950.ts)  
[PlasmaTokensColor--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.349.4193109950.swift)  
[shadow_sbermarket_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.349.4193109950.ts)  
[typo_mage_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.349.4193109950.swift)  
[typo_mage_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.349.4193109950.kt)  
[typo_mage_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.349.4193109950.ts)  
[typo_plasma_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.349.4193109950.swift)  
[typo_plasma_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.349.4193109950.kt)  
[typo_plasma_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.349.4193109950.ts)  
[typo_ruler_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.349.4193109950.swift)  
[typo_ruler_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.349.4193109950.kt)  
[typo_ruler_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.349.4193109950.ts)  
[typo_sage_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.349.4193109950.swift)  
[typo_sage_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.349.4193109950.kt)  
[typo_sage_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.349.4193109950.ts)  
[typo_sbermarket_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.349.4193109950.swift)  
[typo_sbermarket_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.349.4193109950.kt)  
[typo_sbermarket_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.349.4193109950.ts)  
[typo_soulmate_ios-swift--canary.349.4193109950.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.349.4193109950.swift)  
[typo_soulmate_kotlin--canary.349.4193109950.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.349.4193109950.kt)  
[typo_soulmate_react-native--canary.349.4193109950.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.349.4193109950.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.141.1-canary.349.4193109950.0
  npm install @salutejs/plasma-core@1.97.1-canary.349.4193109950.0
  npm install @salutejs/plasma-hope@0.1.1-canary.349.4193109950.0
  npm install @salutejs/plasma-icons@1.125.1-canary.349.4193109950.0
  npm install @salutejs/plasma-temple@1.136.1-canary.349.4193109950.0
  npm install @salutejs/plasma-ui@1.169.1-canary.349.4193109950.0
  npm install @salutejs/plasma-web@1.167.1-canary.349.4193109950.0
  npm install @salutejs/plasma-sb-utils@0.95.1-canary.349.4193109950.0
  # or 
  yarn add @salutejs/plasma-b2c@1.141.1-canary.349.4193109950.0
  yarn add @salutejs/plasma-core@1.97.1-canary.349.4193109950.0
  yarn add @salutejs/plasma-hope@0.1.1-canary.349.4193109950.0
  yarn add @salutejs/plasma-icons@1.125.1-canary.349.4193109950.0
  yarn add @salutejs/plasma-temple@1.136.1-canary.349.4193109950.0
  yarn add @salutejs/plasma-ui@1.169.1-canary.349.4193109950.0
  yarn add @salutejs/plasma-web@1.167.1-canary.349.4193109950.0
  yarn add @salutejs/plasma-sb-utils@0.95.1-canary.349.4193109950.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-349/

File: packages/plasma-web/api/plasma-web.api.md [hunk 1]
```diff
@@ -527,11 +626,20 @@ export const mediaQuery: (breakpoint: Breakpoint, deviceScale?: number) => Media
...
 
-export { Modal }
+// @public
+export const Modal: React_2.FC<ModalProps>;
 
-export { ModalProps }
+// Warning: (ae-forgotten-export) The symbol "ModalViewProps" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export interface ModalProps extends ModalViewProps {
```
Comment (Yeti-or) [inline packages/plasma-web/api/plasma-web.api.md:635]: Кажется это лучше чем было просто с реэкспортами:


https://github.com/salute-developers/plasma/pull/347#discussion_r1108078569

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-349/

File: .lintstagedrc [hunk 2]
```diff
@@ -12,7 +12,6 @@
     ],
     "*.json": [
         "prettier --write --parser json",
-        "node",
```
Comment (Yakutoc) [inline .lintstagedrc:15]: почему убрали?

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-349/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-349/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-349/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-349/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-349/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-349/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-349/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-349/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-349/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-349/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-349/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-349/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-349/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-349/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-349/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-349/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-349/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-349/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-349/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-349/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-349/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-349/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-349/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-349/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-349/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-349/temple-storybook/

**Pull request merged**